.github/workflows/tests.yaml

```diff
@@ -29,6 +29,13 @@ jobs:
           --health-timeout 5s
           --health-retries 5
     steps:
+      - name: Install latest sbom-scorecard
+        run: |
+          mkdir ~/.sbom_scorecard
+          wget -O ~/.sbom_scorecard/scorecard $(curl -s https://api.github.com/repos/ebay/sbom-scorecard/releases/latest | 
```

Comment:
It's generally better to pin a specific release so we know what we're getting. Using latest everytime sets us up to be surprised with a breaking change